src/config/reader.rs

```diff
@@ -72,7 +72,7 @@ impl ConfigReader {
     async fn ext_links(
         &self,
         mut config_module: ConfigModule,
-        path: Option<String>,
+        path: Option<PathBuf>,
```

Comment:
as I understand we changed this behaviour from passing file path to passing directory, right?

at least other naming for argument could be helpful